django_redis/client/default.py

```diff
@@ -59,17 +60,22 @@ def __init__(self, server, params, backend):
 
         self.connection_factory = pool.get_connection_factory(options=self._options)
 
-    def __contains__(self, key):
+    def __contains__(self, key: Any) -> bool:
         return self.has_key(key)
 
-    def get_next_client_index(self, write=True, tried=()):
+    def get_next_client_index(
+        self, write: bool = True, tried: Optional[List[int]] = None
```

Comment:
Should this not be widened to instead?:

```suggestion
        self, write: bool = True, tried: Optional[Collection[int]] = ()
```

Reply:
It should always be a list, if I am not mistaken I purposely changed it because before it could have been tuple and list